scripts/apply.js

```diff
@@ -19,6 +20,11 @@ dotenv.config();
   const pages = await browser.pages();
   await pages[0].close();
 
+  if (!fs.existsSync('.env')) {
+    console.log("File .env not found");
+    process.exit(1);
+  }
+
```

Comment:
Can we add this to scripts/fetchJobs.js aswell?

Reply:
Yes